client/model/files.js

```diff
@@ -388,7 +388,7 @@ class FileSystem{
             const reader = new FileReader();
             reader.readAsText(file);
             reader.onload = () => this.is_binary(reader.result) === false? update_cache(reader.result).then(done) : done();
-            reader.onerror = (err) => err(err);
+            reader.onerror = (_err) => err(_err);
```

Comment:
When I was looking for the other error, I noticed this. I just added `_` just to be sure not to confuse the variable and the function name